demo/src/main/res/layout/activity_message.xml

```diff
@@ -1,110 +1,333 @@
+						android:text="None"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleNone"/>
+			</RadioGroup>
+		</LinearLayout>
+
+		<LinearLayout
+				android:orientation="horizontal"
+				android:layout_width="match_parent"
+				android:layout_height="match_parent">
+
+			<RadioGroup
+					android:layout_width="match_parent"
+					android:layout_height="match_parent"
+					android:orientation="horizontal"
+					android:paddingBottom="16dp"
+					android:id="@+id/colorRadioGroup">
+
+				<RadioButton
+						android:text="Black"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleBlack" android:checked="true"/>
+				<RadioButton
+						android:text="White"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleWhite"/>
+				<RadioButton
+						android:text="Monochrome"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleMonochrome"/>
+				<RadioButton
+						android:text="Grayscale"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleGrayscale"/>
+
+			</RadioGroup>
+
+		</LinearLayout>
+
+		<LinearLayout
+				android:orientation="horizontal"
+				android:layout_width="match_parent"
+				android:layout_height="match_parent">
+
+			<RadioGroup
+					android:layout_width="match_parent"
+					android:layout_height="match_parent"
+					android:orientation="horizontal"
+					android:paddingBottom="16dp"
+					android:id="@+id/alignmentRadioGroup">
+				<RadioButton
+						android:text="Left"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleLeft" android:checked="true"/>
+				<RadioButton
+						android:text="Center"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleCenter"/>
+				<RadioButton
+						android:text="Right"
+						android:layout_width="wrap_content"
+						android:layout_height="wrap_content"
+						android:id="@+id/styleRight"/>
+			</RadioGroup>
+
+		</LinearLayout>
+
+		<TextView
+				android:text="Offer Type"
+				android:layout_width="match_parent"
+				android:layout_height="wrap_content"
+				android:id="@+id/offerTypeLabel"
+				android:paddingTop="16dp"
+				android:paddingBottom="16dp"
+				android:textStyle="bold"
+				android:textSize="16sp"/>
+
+		<LinearLayout
+				android:orientation="horizontal"
+				android:layout_width="match_parent"
+				android:layout_height="match_parent">
+			<ToggleButton
+					android:text="Short Term"
+					android:layout_width="wrap_content"
+					android:layout_height="wrap_content" android:id="@+id/shortTerm" android:layout_weight="1"
+					android:textSize="10sp"
+					android:textOn="Short Term" android:textOff="Short Term"/>
+			<ToggleButton
```

Comment:
Why buttons instead of a radio group?

Reply:
With a radio inputs there is no way to deselect an offer type and could never get the generic message.